packages/devextreme/js/__internal/ui/html_editor/html_editor.ts

```diff
@@ -575,14 +577,25 @@ class HtmlEditor extends Editor<Properties> {
     this._formDialog = new FormDialog(this.$element(), options);
   }
 
-  _renderAIDialog(): void {
-    const { aiIntegration } = this.option();
+  _shouldRenderAIDialog(): boolean {
+    const { aiIntegration, toolbar } = this.option();
 
-    if (!aiIntegration) {
-      return;
+    if (!(aiIntegration && toolbar?.items)) {
```

Comment:
```suggestion
    if (!aiIntegration || !toolbar?.items) {
```

Will be easier to read for negations

Reply:
Zhenya Kiyashko would disagree with you 🤣